pkgs/by-name/lo/localstack/package.nix

```diff
@@ -1,19 +1,22 @@
 {
+  fetchFromGitHub,
   lib,
+  localstack,
+  nix-update-script,
   python3,
-  fetchFromGitHub,
+  testers,
 }:
 
 python3.pkgs.buildPythonApplication rec {
```

Comment:
It would be nice if you could change this to the more apropriate `python3Packages`:
- https://github.com/NixOS/nixpkgs/issues/211340